notebooks/youtube_transcript_search.ipynb

```diff
@@ -28,7 +28,7 @@
     }
    ],
    "source": [
-    "pip install --quiet openai tqdm ratelimiter retry"
+    "pip install --quiet openai tqdm ratelimiter retry datasets lance"
```

Comment:
the pypi package is named `pylance` (lance was already taken unfortunately).

Reply:
ok, it's corrected to pylance